gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -663,14 +663,9 @@ case class ColumnarOverrideRules(session: SparkSession)
   // while creating the rules. At this time SQLConf may not be there yet.
 
   def preOverrides(): List[SparkSession => Rule[SparkPlan]] = {
-    val tagBeforeTransformHitsRules = if (this.isAdaptiveContext) {
-      TagBeforeTransformHits.ruleBuilders
-    } else {
-      // When AQE is supported, rules are applied in ColumnarQueryStagePrepOverrides
-      List.empty
```

Comment:
this is wrong, if isAdaptiveContext is true that means we are in AQE...

Reply:
Oops but nice catch!